common/djangoapps/oauth_exchange/tests/utils.py

```diff
@@ -101,6 +69,12 @@ def test_confidential_client(self):
             "test_client_id is not a public client"
         )
 
+    def test_inactive_user(self):
+        self.user.is_active = False
+        self.user.save()
+        self._setup_provider_response(success=True)
+        self._assert_success(self.data, expected_scopes=[])
```

Comment:
Yes, that's correct.
And I didn't want to automatically set the "is_active" flag (that currently means is_email_verified) when someone registers via Google/FB.